### PR TITLE
[zh] Translate dual-stack-support page

### PR DESCRIPTION
**Pull request by tengqm**
/language zh

### CODE REVIEW

File: content/zh/docs/setup/production-environment/tools/kubeadm/dual-stack-support.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+```yaml
+apiVersion: kubeadm.k8s.io/v1beta3
+kind: JoinConfiguration
+discovery:
+  bootstrapToken:
+    apiServerEndpoint: 10.100.0.1:6443
+    token: "clvldh.vjjwg16ucnhp94qr"
+    caCertHashes:
+    - "sha256:a4863cde706cfc580a439f842cc65d5ef112b7b2be31628513a9881cf0d9fe0e"
+    # change auth info above to match the actual token and CA certificate hash for your cluster
```
Comment (sftim) [inline content/zh/docs/setup/production-environment/tools/kubeadm/dual-stack-support.md:160]: nit: you could localize this comment too

File: content/zh/docs/setup/production-environment/tools/kubeadm/dual-stack-support.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+  localAPIEndpoint:
+    advertiseAddress: "10.100.0.2"
+    bindPort: 6443
+discovery:
+  bootstrapToken:
+    apiServerEndpoint: 10.100.0.1:6443
+    token: "clvldh.vjjwg16ucnhp94qr"
+    caCertHashes:
+    - "sha256:a4863cde706cfc580a439f842cc65d5ef112b7b2be31628513a9881cf0d9fe0e"
+    # change auth info above to match the actual token and CA certificate hash for your cluster
```
Comment (sftim) [inline content/zh/docs/setup/production-environment/tools/kubeadm/dual-stack-support.md:186]: nit: you could localize this comment too

**Review by sftim — COMMENTED** [review]
Some nits / suggestions

File: content/zh/docs/setup/production-environment/tools/kubeadm/dual-stack-support.md (hunk 2 above)
Comment (tengqm) [reply content/zh/docs/setup/production-environment/tools/kubeadm/dual-stack-support.md]: Thanks.

**Pull request merged**